packages/nimble-components/src/select/index.ts

```diff
@@ -49,51 +146,689 @@ export class Select extends FoundationSelect implements ErrorPattern {
     @observable
     public hasOverflow = false;
 
-    // Workaround for https://github.com/microsoft/fast/issues/5123
-    public override setPositioning(): void {
+    /**
+     * @internal
+     */
+    @observable
+    public filteredOptions: ListboxOption[] = [];
+
+    /**
+     * @internal
+     */
+    @observable
+    public filter = '';
+
+    /**
+     * @internal
+     */
+    @observable
+    public scrollbarIsVisible = false;
+
+    /**
+     * @internal
+     */
+    @observable
+    public committedSelectedOption: ListboxOption | undefined = undefined;
+
+    /**
+     * The max height for the listbox when opened.
+     *
+     * @internal
+     */
+    @observable
+    public maxHeight = 0;
+
+    /**
+     * The component is collapsible when in single-selection mode with no size attribute.
+     *
+     * @internal
+     */
+    @volatile
+    public get collapsible(): boolean {
```

Comment:
It's possible that we could remove this member as it's intended to make the `Select` look more like a `Listbox`.

Reply:
I was about to vote to remove it but it looks like it's referenced a lot so probably not worth the effort to do so in this PR.